website/docs/reference/global-configs/legacy-behaviors.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Legacy behaviors"
+id: "legacy-behaviors"
+sidebar: "Legacy behaviors"
+---
+
+Most flags exist to configure runtime behaviors with multiple valid choices. The right choice may vary based on the environment, user preference, or the specific invocation.
+
+Another category of flags provides existing projects with a migration window for runtime behaviors that are changing in newer releases of dbt. These flags help us achieve a balance between these goals, which can otherwise be in tension:
```

Comment:
```suggestion
Another category of flags provides existing projects with a migration window for runtime behaviors that are changing in newer releases of dbt. These flags help us achieve a balance between these goals, which can otherwise be in tension, by:
```